src/Aquifer.Public.API/OpenApi/SwaggerDocumentSettings.cs

```diff
@@ -48,6 +55,7 @@ All endpoints require an API key in the `api-key` header.<br><br>
                         In = OpenApiSecurityApiKeyLocation.Header,
                         Type = OpenApiSecuritySchemeType.ApiKey
                     });
+                ds.MarkNonNullablePropsAsRequired();
```

Comment:
Unfortunately the Kiota client uses nullable properties everywhere, even if the property is required.  Fixing this is their most upvoted issue: https://github.com/microsoft/kiota/issues/3911.  Adding this FastEndpoints setting at least gets us a correct OpenApi JSON spec and whenever Kiota gets a v2 release we should be able to take advantage of non-nullable reference types in the generated request/response objects.